### PR TITLE
Azure Docs & Azure settings.

### PR DESCRIPTION
**Pull request by razanfawwaz**
## What kind of change does this PR introduce?

Some improvement from PR #7882

Docs:
Change the Azure tenant URL heading size, from h2 to h3 like the others.

Dashboard:
Add help message to using the Value not the Secret ID on Azure Secret.
## What is the current behavior?

Please link any relevant issues here.
The wrong Value in Secret ID can make the error as mentioned in #1192 (Unable to exchange external code).

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/stores/jsonSchema/auth_gotrue_config.json [hunk 1]
```diff
@@ -113,7 +113,8 @@
     },
     "EXTERNAL_AZURE_SECRET": {
       "title": "Azure secret",
-      "type": "string"
+      "type": "string",
+      "help": "Enter the data from Value, not the Secret ID."
```
Comment (J0) [inline studio/stores/jsonSchema/auth_gotrue_config.json:117]: Content wise it looks good to me -- not sure if the help json has an effect for the secrets in Auth settings -- will let someone from frontend confirm  before merging.

**Review by J0 — APPROVED** [review]

**Review by razanfawwaz — COMMENTED** [review]
I think this is the right file @J0

File: studio/stores/jsonSchema/auth_gotrue_config.json (hunk 1 above)
Comment (razanfawwaz) [reply studio/stores/jsonSchema/auth_gotrue_config.json:117]: I add the description on `AuthProvidersFormValidtaion.tsx` maybe you can check my new commit.

**Comment by razanfawwaz** [comment]
@supabase/frontend

**Comment by razanfawwaz** [comment]
Hi @inian! maybe you can check this PR! thank you!

**Comment by razanfawwaz** [comment]
@kiwicopple @inian

**Review by kiwicopple — APPROVED** [review]

**Comment by kiwicopple** [comment]
Sorry for the delay @razanfawwaz - this looks great! I've approved it, ready for merging but there are some merge-conflicts. Would you mind resolving them and then we can get this into prod?

**Comment by razanfawwaz** [comment]
hi! @kiwicopple i'll try to fix the conflict! thank you!

**Review by kiwicopple — APPROVED** [review]

**Comment by razanfawwaz** [comment]
hi @kiwicopple i think I've made some mistakes in this pull request, so I create a new pr in #9166 
thanks!

**Pull request closed without merging**